editor.planx.uk/src/@planx/components/List/utils.tsx

```diff
@@ -39,7 +39,7 @@ export function formatSchemaDisplayValue(
     }
     case "question": {
       const matchingOption = field.data.options.find(
-        (option) => option.data.text === value,
+        (option) => option.data.text === value || option.data.val === value,
```

Comment:
this is a small bug fix following on from https://github.com/theopensystemslab/planx-new/pull/3521 spotted while testing the List component locally

we need to account for "question" fields that set a `val` OR only `text` - our previous fixes only accounted for one or the other!